### ISSUE TITLE
Straight planes get dents when converting to mesh

### ISSUE THREAD

**Issue by jclarkk** [3DTopia/LGM#20]
It seems that when I run the convery.py on an asset that has straight planes I encounter dents such as this:
![image](https://github.com/3DTopia/LGM/assets/6411191/8b0b4673-6c91-4b21-b22e-610bbef242ce)

Any idea how we might be able to overcome this?

Thanks!

**Comment by jclarkk** [comment]
It seems restoring laplacian_smooth_loss has helped a lot but still not perfect.

Current results:
![image](https://github.com/3DTopia/LGM/assets/6411191/bda95e7b-a4fb-4aca-aa58-33e64dc742c7)

Here is how I restored it:
`            loss_mse = F.mse_loss(image_pred, image_gt) + 0.1 * F.mse_loss(alpha_pred, alpha_gt)
            loss_lap = laplacian_smooth_loss(self.v + self.deform, self.f)
            loss_normal = normal_consistency(self.v + self.deform, self.f)
            loss_offsets = (self.deform ** 2).sum(-1).mean()
            loss = loss_mse + 0.001 * loss_normal + 0.1 * loss_offsets * loss_lap * 0.01`

**Comment by ashawkey** [comment]
@jclarkk Thanks for the information!
